src/enums/viewState.ts

```diff
@@ -1,6 +1,14 @@
 // main
 import { makeEnum } from '@transcend-io/type-utils';
 
+/**
+ * Consent Manager view state for the Interactive Advertising Bureau
+ */
+export const IABViewState = makeEnum({
+  /* Transparency and Consent Framework for Europe */
+  TCF_EU: 'TCF_EU',
```

Comment:
Sorry but can we align with the CamelCase like all the other view states actually?

Reply:
Hm. The problem with CamelCase is these are both acronyms. So, technically, CamelCase would be `TCFEU` which is not very readable. Of course, many might think `TcfEu` would be correct. I would not be among them.

Reply:
i'm cool with TCFEU